packages/selection/frontend/sirius-components-selection/src/SelectionDialogTreeView.tsx

```diff
@@ -0,0 +1,89 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { TreeItemActionProps, TreeView } from '@eclipse-sirius/sirius-components-trees';
+import UnfoldMoreIcon from '@mui/icons-material/UnfoldMore';
+import IconButton from '@mui/material/IconButton';
+import { useState } from 'react';
+import { makeStyles } from 'tss-react/mui';
+import { SelectionDialogTreeViewProps, SelectionDialogTreeViewState } from './SelectionDialogTreeView.types';
+import { useSelectionDialogTreeSubscription } from './useSelectionDialogTreeSubscription';
+export const SELECTION_DIALOG_TYPE: string = 'selectionDialogDescription';
+
+const useTreeStyle = makeStyles()((theme) => ({
+  borderStyle: {
+    border: '1px solid',
+    borderColor: theme.palette.grey[500],
```

Comment:
theme.palette.divider to stay with the same style as the rest of the application

Reply:
Done